index.js

```diff
@@ -5,7 +5,7 @@ const console = require('console')
 
 const DISPLAY_NAME_SYMBOL = Symbol.for('fastify.display-name')
 const fpStackTracePattern = new RegExp('at\\s{1}plugin\\s{1}.*\\n\\s*(.*)')
-const fileNamePattern = new RegExp('\\/(\\w*)\\.js')
+const fileNamePattern = new RegExp('\\/(\\w*)(\\.\\w*)?\\.js')
```

Comment:
I would prefer if we include hits in the previous group, basically "anything that is not a /". Note that the file name could also include numbers.

Can you add a unit test?

Reply:
If a file is named: `myPlugin123.whatever.js`
Should the appropriate `'fastify.display-name'` be: `myPlugin123.whatever` or `myPlugin123`?

Reply:
> A unit test would require specially named files. Is that okay?

Yes.

> If a file is named: myPlugin123.whatever.js
Should the appropriate 'fastify.display-name' be: myPlugin123.whatever or myPlugin123?

it should be `myplugin123.whatever`.

Reply:
Great! Will do.